apps/changelog/classes/change_log.py

```diff
@@ -0,0 +1,82 @@
+# Python Standard Library Imports
+from collections import namedtuple
+
+
+# isort: off
+
+
+class ChangeLog(
+    namedtuple(
+        'ChangeLog',
+        'origin_url,log_entries',
+    )
+):
+    # origin_url: str
+    # log_entries: list[LogEntry]
+
+    def write_changelog(
+        self,
+        changelog_file_name,
+        slack_channel,
```

Comment:
Please invert the ordering and make `slack_channel=None` by default. If `slack_announce=True`, check that `slack_channel` is also set, and if not, raise Exception / show error / exit early